eks/weather/helm/templates/service.yaml

```diff
@@ -1,23 +1,25 @@
+{{- $ings := .Values.ingresses | default (list) -}}
```

Comment:
This assumes we only do ingresses, we want to offer a way to use service with no ingress, user should be able to create the service, and optional configure ingress